vpc/service/assign_addresses_v3.go

```diff
@@ -618,26 +601,26 @@ func (vpcService *vpcService) assignIPsToENI(ctx context.Context, req *vpcapi.As
 		}
 	case *vpcapi.AssignIPRequestV3_TransitionRequested:
 		resp.TransitionAssignment = &vpcapi.AssignIPResponseV3_TransitionAssignment{
-			AssignmentId: transitionAss.assignmentID,
+			AssignmentId: transitionAss.TaskID,
```

Comment:
I'm really worried that assignment id and taskID are getting mixed here

Reply:
You're right. It is indeed a bit special when it comes to transition assignment. The assignment ID is not exactly the task ID but a generated ID in the form of "t-<UUID>", calling it `taskID` is confusing. Will revert this.